routes/permafrost.py

```diff
@@ -92,43 +98,82 @@ def package_obu_vector(obu_vector_resp):
         summary_operation -- (str) one of 'min', 'avg', or 'max'
     Returns:
         gipl1km_wcps_str -- (str) fragment used to construct the WCPS request
     """
-    gipl1km_wcps_str = quote(
-        (
-            f"ProcessCoverages&query=for $c in (crrel_gipl_outputs) "
-            f"  return encode (coverage summary over $v variable(0:9)"
-            f"  values {summary_operation}( $c[variable($v),year({years}),X({x}),Y({y})] )"
-            f', "application/json")'
+    if model is None and scenario is None:
+        gipl1km_wcps_str = quote(
+            (
+                f"ProcessCoverages&query=for $c in (iem_gipl_historical) "
+                f"  return encode (coverage summary over $v variable(0:8)"
+                f"  values {summary_operation}( $c[variable($v),year({years}),X({x}),Y({y})] )"
+                f', "application/json")'
+            )
         )
-    )
-    return gipl1km_wcps_str
+        return gipl1km_wcps_str
+    else:
+        gipl1km_wcps_str = quote(
+            (
+                f"ProcessCoverages&query=for $c in (crrel_gipl_outputs) "
+                f"  return encode (coverage summary over $v variable(0:9)"
+                f"  values {summary_operation}( $c[variable($v),model({model}),scenario({scenario}),year({years}),X({x}),Y({y})] )"
+                f', "application/json")'
+            )
+        )
+        return gipl1km_wcps_str
 
 
-def package_gipl1km_wcps_data(gipl1km_wcps_resp):
-    """Package a min-mean-max summary of ten GIPL 1 km variables for a given year range. Values are rounded to one decimal place because units are either meters or degrees C.
+def package_gipl1km_wcps_data(gipl1km_wcps_resp, package_type):
+    """Package a min-mean-max summary of ten GIPL 1 km variables for a given year range and model type. Values are rounded to one decimal place because units are either meters or degrees C.
 
     Arguments:
-        gipl1km_wcps_resp -- (list) nested 2-level list of the WCPS response values. The response order must be min-mean-max: [[min], [mean], [max]]
-
+        gipl1km_wcps_resp -- (list) nested 3-level list of the WCPS response values. The response order must be min-mean-max: [[min], [mean], [max]]
+        package_type -- (string) either historical or projected
     Returns:
         gipl1km_wcps_point_pkg -- (dict) min-mean-max summarized results for all ten variables
     """
-    gipl1km_wcps_point_pkg = {}
+    gipl1km_wcps_point_pkg = dict()
     summary_methods = ["min", "mean", "max"]
-    for resp, stat_type in zip(gipl1km_wcps_resp, summary_methods):
-        gipl1km_wcps_point_pkg[f"gipl1km{stat_type}"] = {}
-        for k, v in zip(gipl1km_dim_encodings["variable"].values(), resp):
-            gipl1km_wcps_point_pkg[f"gipl1km{stat_type}"][k] = round(v, 1)
+    if package_type == "historical":
+        gipl1km_wcps_point_pkg["Historical"] = dict()
+        gipl1km_wcps_point_pkg["Historical"]["Historical"] = dict()
```

Comment:
I'm wondering if one of these dict keys should be "CRU-TS 4.0" to indicate the "model" so-to-speak used for the historical baseline?

Reply:
Yes, you are correct that we should specify the model as CRU-TS 4.0 and the scenario as Historical. Thanks for catching this! I've added that change to the most recent commit.